api/graphql/v2/users.graphqls

```diff
@@ -0,0 +1,24 @@
+scalar Time
+scalar UUID
+
+extend type Query  {
+    me: User!
+}
+
+type User {
+    id: UUID!
+    filmstadenMembershipId: String
+    name: String!
+    firstName: String!
+    lastName: String!
+    nick: String
+    email: String!
+    phone: String
+    avatarURL: String
+#    giftCertificates: [GiftCertificateDTO!]!
```

Comment:
Check: How come these are ignored for now?

Reply:
not implemented yet (requires another table)